api/PclusterApiHandler.py

```diff
@@ -533,15 +534,23 @@ def get_instance_types():
     return {"instance_types": sorted(instance_types, key=lambda x: x["InstanceType"])}
 
 
+def _get_user_roles(decoded):
+    return decoded[USER_ROLES_CLAIM] if USER_ROLES_CLAIM in decoded else ["user"]
+
+
+
 def get_identity():
```

Comment:
issue(blocking): I'd take this chance to add our first test. It can be just an happy path test where we test that the function is returning the expected `decoded` object